WordPress/twentytwentyone
# Try/menu descriptions

Fixes #414 

## Summary
Adds the description to menu items.

In the screenshot I'm including items with and without descriptions as well as short/long titles/descriptions to get an idea of the alignments.

![Screenshot_2020-10-12 Lorem Ipsum – My Blog](https://user-images.githubusercontent.com/588688/95764287-eb87e680-0cb8-11eb-8ae1-3ca4ef625b05.png)

## Test instructions

This PR can be tested by following these steps:
1. Add descriptions to menu items
1. Check out the front-end.

## Quality assurance
